autoload/neomake.vim

```diff
@@ -512,7 +512,7 @@ function! s:AddExprCallback(maker) abort
             call setqflist(list, 'r')
         endif
     endif
-    if counts_changed
+    if counts_changed && has_key(a:maker, 'bufnr')
         call s:neomake_hook('NeomakeCountsChanged', {
                     \ 'file_mode': a:maker.file_mode,
                     \ 'bufnr': a:maker.bufnr})
```

Comment:
I think the proper fix here would be to not pass in `a:maker.bufnr` to the hook, but still call it (since counts changed).
Would be better of in a separate PR.